### PR TITLE
docs: #1241 clarify --ips flag requirements

### LINKED ISSUE #1241: clarify requirements for setting --ips flag in rpk redpanda config bootstrap
### Describe the Issue

See https://redpandadata.slack.com/archives/C0246LPDYDU/p1676471989966239
takeaway is we should belabor the rules for setting --ips in that rpk command

SME = Andrew Wong, Chris Larsen

### Updates to existing documentation

https://docs.redpanda.com/docs/deploy/deployment-option/self-hosted/manual/production/production-deployment/#start-redpanda

### PR DESCRIPTION
**Pull request by micheleRP**
resolves #1241 

preview: https://deploy-preview-1242--redpanda-documentation.netlify.app/docs/deploy/deployment-option/self-hosted/manual/production/production-deployment/#start-redpanda

### CODE REVIEW

**Comment by micheleRP** [comment]
approved by Chris: https://redpandadata.slack.com/archives/C0246LPDYDU/p1676482540616759?thread_ts=1676471989.966239&cid=C0246LPDYDU

File: docs/deploy/deployment-option/self-hosted/manual/production/production-deployment.mdx [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: Deploy for Production
 
 <head>
     <meta name="title" content="Deploy for Production | Redpanda Docs"/>
-    <meta name="description" content="Steps to deploy Redpanda in production."/>
+    <meta name="description" content="Steps to deploy a production cluster in Redpanda"/>
```
Comment (shorgi) [inline docs/deploy/deployment-option/self-hosted/manual/production/production-deployment.mdx:7]:
```suggestion
    <meta name="description" content="Steps to deploy a Redpanda production cluster"/>
```

File: versioned_docs/version-22.3/deploy/deployment-option/self-hosted/manual/production/production-deployment.mdx [hunk 2]
```diff
@@ -4,7 +4,7 @@ title: Deploy for Production
 
 <head>
     <meta name="title" content="Deploy for Production | Redpanda Docs"/>
-    <meta name="description" content="Steps to deploy Redpanda in production."/>
+    <meta name="description" content="Steps to deploy a production cluster in Redpanda."/>
```
Comment (shorgi) [inline versioned_docs/version-22.3/deploy/deployment-option/self-hosted/manual/production/production-deployment.mdx:7]:
```suggestion
    <meta name="description" content="Steps to deploy a Redpanda production cluster."/>
```

File: docs/deploy/deployment-option/self-hosted/manual/production/production-deployment.mdx [hunk 3]
```diff
@@ -139,8 +139,12 @@ sudo rpk redpanda config set redpanda.empty_seed_starts_cluster false && \
 sudo systemctl start redpanda-tuner redpanda
 ```
 
-- The `--self` flag tells Redpanda to which interface address to bind. Usually this is its private IP.
-- The `--ips` flag lists all the seed nodes in the cluster (seed nodes correspond to the `seed_servers` property in `redpanda.yaml`).
+- The `--self` flag tells Redpanda the interface address to bind to. Usually this is its private IP.
+- The `--ips` flag lists all the seed nodes in the cluster, including the one being started. Seed nodes correspond to the `seed_servers` property in `redpanda.yaml`. 
+
+  :::note
+  The `--ips` flag must be set *identically* (with nodes listed in identical order) on each node.
```
Comment (shorgi) [inline docs/deploy/deployment-option/self-hosted/manual/production/production-deployment.mdx:146]:
```suggestion
  The `--ips` flag must be set *identically* on each node, with nodes listed in identical order.
```

File: versioned_docs/version-22.3/deploy/deployment-option/self-hosted/manual/production/production-deployment.mdx [hunk 4]
```diff
@@ -139,8 +139,12 @@ sudo rpk redpanda config set redpanda.empty_seed_starts_cluster false && \
 sudo systemctl start redpanda-tuner redpanda
 ```
 
-- The `--self` flag tells Redpanda to which interface address to bind. Usually this is its private IP.
-- The `--ips` flag lists all the seed nodes in the cluster (seed nodes correspond to the `seed_servers` property in `redpanda.yaml`).
+- The `--self` flag tells Redpanda the interface address to bind to. Usually this is its private IP.
+- The `--ips` flag lists all the seed nodes in the cluster, including the one being started. Seed nodes correspond to the `seed_servers` property in `redpanda.yaml`. 
+
+  :::note
+  The `--ips` flag must be set *identically* (with nodes listed in identical order) on each node.
```
Comment (shorgi) [inline versioned_docs/version-22.3/deploy/deployment-option/self-hosted/manual/production/production-deployment.mdx:146]:
```suggestion
  The `--ips` flag must be set *identically* on each node, with nodes listed in identical order.
```

**Review by shorgi — APPROVED** [review]
Approved, see suggestions

**Pull request merged**